addons/sourcemod/scripting/nt_classlimit.sp

```diff
@@ -122,11 +122,40 @@ public void OnPluginStart()
 		HookClassSelectionPfns(client);
 		break;
 	}
-
+	
+	RegAdminCmd("sm_classlimit", Command_Limit, ADMFLAG_GENERIC);
+	
 	// Create the default config file, if it doesn't exist yet
 	AutoExecConfig();
 }
 
+public Action Command_Limit(int client, int args)
+{
+	if (args == 0)
+	{
+		ReplyToCommand(client, "Usage !classlimit x where x sets the limit for all classes");
+		return Plugin_Handled;
+	}
+	
+	char limitArg[2 + 1];
+	GetCmdArg(1, limitArg, sizeof(limitArg));
+	int limit = StringToInt(limitArg);
+
+	if(limit == 0)
+	{
+		limit = 16;
+		ReplyToCommand(client, "Setting limit failed, setting limits to 16 each");
+	}
```

Comment:
Should be `if (limit <= 0)` so negative values can't be used. You could also consider refusing to change the value for invalid inputs here instead of defaulting to 16.

Reply:
True.